Update package.nix

IliyanKostov9 opened the pull request:
Added ffmpeg dependency for viber (opening media in viber fails, due to missing ffmpeg) 

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

tobim committed:
ledger: fix build with boost 1.86

toastal committed:
use movim’s database user

toastal committed:
use mkDefaults for Nginx config

toastal committed:
typo in script name

toastal committed:
syntax hints for tree-sitter

toastal committed:
optionally add group to nginx user

Pandapip1 committed:
oscavmgr: add version test

toastal committed:
nixos/movim: rm logs if found

toastal committed:
movim: make mainProgram name match

Bot-wxt1221 committed:
gpufetch: init at 0.25

Bot-wxt1221 committed:
peakperf: init at 1.17-unstable-2024-10-07

DimitarNestorov committed:
swiftformat: 0.47.10 -> 0.55.4

Samasaur1 committed:
xcodegen: init

r-ryantm committed:
plexRaw: 1.41.3.9292-bc7397402 -> 1.41.3.9314-a0bfb8370

khaneliman committed:
luaPackages: update on 2024-12-24

khaneliman committed:
vimPlugins: update on 2024-12-24

khaneliman committed:
vimPlugins.nvim-treesitter: update grammars

ck3d committed:
x32edit,m32edit: 4.1 -> 4.3

numinit committed:
teams: create Android team

We have needed this for a while, especially as it relates to
coordinating Android repo updates.

Create it with the people who expressed interest in:
https://github.com/NixOS/nixpkgs/issues/359918

fpletz committed:
sofia-sip: fix build with gcc-14

adrian-gierakowski committed:
maintainers: add adrian-gierakowski

jtojnar committed:
gegl: 0.4.50 → 0.4.52

https://gitlab.gnome.org/GNOME/gegl/-/compare/GEGL_0_4_50...GEGL_0_4_52

jpteb committed:
stlink: patch incorrect calloc argument order

With the update to GCC 14, stlink v1.8.0 is broken, as GCC 14 enforces
the correct argument order of calloc. This commit adds a little patch to
resolve that issue for now. This patch should be removed and stlink
updated to v1.8.1 as soon as that version gets released.

4r7if3x committed:
treegen: init at 1.1.0

nevivurn committed:
libdbiDrivers: fix compile errors in tests, backport patches

wineee committed:
lucenepp: fix build with boost 1.85.0

dywedir committed:
broot: 1.44.2 -> 1.44.3 (#368454)

wineee committed:
dmraid: patch dmevent_tool.c to compile on modern gcc versions (#368470)

trofi committed:
vcmi: 1.6.0 -> 1.6.1

Changes: https://github.com/vcmi/vcmi/releases/tag/1.6.1

wineee committed:
deepin.deepin-system-monitor: fix build with gcc 14 (#368741)

r-ryantm committed:
ananicy-rules-cachyos: 0-unstable-2024-12-17 -> 0-unstable-2024-12-27

r-ryantm committed:
zapret: 69.8 -> 69.9

linsui committed:
msgpack-cxx: fix build

symphorien committed:
nixos/sane: ensure saned can access usb scanners (#367198)

xaverdh committed:
oneko: fix build by disabling some compiler warnings

This is old code, that will not change any time soon.

GaetanLepage committed:
uv: 0.5.12 -> 0.5.13

Diff: https://github.com/astral-sh/uv/compare/None...0.5.13

Changelog: https://github.com/astral-sh/uv/blob/0.5.13/CHANGELOG.md

fabaff committed:
python312Packages.tencentcloud-sdk-python: 3.0.1290 -> 3.0.1291 (#368645)

fabaff committed:
python312Packages.pynordpool: 0.2.3 -> 0.2.4 (#368529)

fabaff committed:
python312Packages.bloodyad: 2.1.5 -> 2.1.7 (#368551)

r-ryantm committed:
inspircd: 3.17.1 -> 3.18.0

nixpkgs-merge-bot[bot] committed:
opnborg: 0.1.49 -> 0.1.51 (#368737)

misuzu committed:
libfm: fix build (#368590)

GaetanLepage committed:
nixos/zfs-replication: fix typo

r-ryantm committed:
code-cursor: 0.44.5 -> 0.44.9

GaetanLepage committed:
nixos/zfs-replication: fix typo (#368764)

K900 committed:
python312Packages.debugpy: fix hash due to .gitattributes (#368643)

thiagokokada committed:
nixos-rebuild-ng: refactors (#368619)

misuzu committed:
jamesdsp: fix build with gcc 14 (#368725)

misuzu committed:
lxsession: fix build with gcc 14 (#368711)

dotlambda committed:
beamerpresenter: 0.2.5 -> 0.2.6 (#368666)

GaetanLepage committed:
uv: 0.5.12 -> 0.5.13 (#368761)

NetaliDev committed:
taschenrechner: 1.4.0 -> 1.5.0 (#368689)

rnhmjoj committed:
nixos/networkmanager: match upstream units Install sections

The upstream NetworkManager.service unit[1] has this Install section:

  [Install]
  WantedBy=multi-user.target
  Also=NetworkManager-dispatcher.service
  Also=NetworkManager-wait-online.service

Let's just match that in NixOS and don't try to mess with
network.target. The latter should only be used to order units at
shutdown, not when booting.

[1]: https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/blob/main/data/NetworkManager.service.in?ref_type=heads

GaetanLepage committed:
ruff: Build as a standalone without Python (#358029)

4r7if3x committed:
godini: init at 1.0.0

rnhmjoj committed:
nixos/tests/networkmanager: test NM is started at boot

Shawn8901 committed:
proton-ge-bin: GE-Proton9-21 -> GE-Proton9-22

thiagokokada committed:
pamtester: add autoreconfHook

Fix build with GCC 14.

sikmir committed:
devcontainer: 0.71.0 → 0.72.0, fix build (#368516)

ulrikstrid committed:
ocamlPackages.httpun-ws-lwt: init at 0.2.0 (#366950)

xanderio committed:
spice-gtk: fix darwin build (#368148)

xanderio committed:
cargo-cache: fix compilation on darwin (#368717)

markuskowa committed:
dep-tree: 0.23.0 -> 0.23.1 (#368112)

thiagokokada committed:
libretro.mupen64plus: add patches for GCC14

Scrumplex committed:
perlPackages.NetIDNEncode: fix use of function removed in Perl 5.38.0

Signed-off-by: Sefa Eyeoglu <contact@scrumplex.net>

JohnRTitor committed:
ananicy-rules-cachyos: 0-unstable-2024-12-17 -> 0-unstable-2024-12-27 (#368756)

misuzu committed:
rip2: set up updateScript (#367987)

emilazy committed:
heaptrack: add patch for GCC 14, unpin Boost (#358970)

r-ryantm committed:
vdrPlugins.markad: 4.2.7 -> 4.2.8

ulrikstrid committed:
ocamlPackages.gluten-mirage: init at 0.5.2 (#366942)

ulrikstrid committed:
ocamlPackages.gluten-async: init at 0.5.2 (#366943)

nixpkgs-merge-bot[bot] committed:
txr: 297 -> 298 (#368552)

thiagokokada committed:
uasm: switch to gcc13Stdenv

thiagokokada committed:
uasm: 2.56.2 -> 2.57

donovanglover committed:
bacon: 3.5.0 -> 3.6.0 (#367637)

JohnRTitor committed:
equicord: 1.10.6 -> 1.10.8; add update script (#364064)

flokli committed:
ledger: fix build with boost 1.86 (#359441)

flokli committed:
hplip: fix build (#368507)

r-ryantm committed:
buildpack: 0.36.1 -> 0.36.2

ck3d committed:
vdrPlugins.markad: 4.2.7 -> 4.2.8 (#368801)

flokli committed:
tracy: 0.10 -> 0.11.1 + other changes (#328253)

nixpkgs-merge-bot[bot] committed:
zapret: 69.8 -> 69.9 (#368757)

aanderse committed:
kodiPackages.keymap: 1.2.1 -> 1.3.0 (#367616)

r-ryantm committed:
clusternet: 0.17.2 -> 0.17.3

r-ryantm committed:
orbiton: 2.68.5 -> 2.68.6

r-ryantm committed:
home-assistant-custom-lovelace-modules.hourly-weather: 6.4.0 -> 6.5.0

r-ryantm committed:
nchat: 5.3.5 -> 5.4.2

r-ryantm committed:
pik: 0.12.0 -> 0.13.0

flokli committed:
magic-wormhole: fix test for twisted 24.10.0 (#367669)

azahi committed:
vcmi: 1.6.0 -> 1.6.1 (#368753)

donovanglover committed:
glance: 0.6.3 -> 0.6.4 (#368444)

haskelious committed:
quartus-prime-lite: 23.1std.0.991 -> 23.1std.1.993

adamcstephens committed:
pam_ssh_agent_auth: fix build with gcc14 (#368612)

Aleksanaa committed:
msgpack-cxx: fix build (#368758)

Aleksanaa committed:
oneko: fix build by disabling some compiler warnings (#368759)

misuzu committed:
pcmanfm: fixup build with gcc14

Defelo committed:
cargo-semver-checks: 0.34.0 -> 0.38.0

nixpkgs-merge-bot[bot] committed:
buildpack: 0.36.1 -> 0.36.2 (#368809)

FliegendeWurst committed:
fzf-tab-zsh: fix build with gcc14 (#368738)

Defelo committed:
cargo-semver-checks: refactor

jian-lin committed:
rl-2411: add an entry for the new kmonad module

r-ryantm committed:
cargo-show-asm: 0.2.43 -> 0.2.45

nixpkgs-merge-bot[bot] committed:
aliae: 0.23.0 -> 0.24.1 (#368681)

emilazy committed:
jxrlib: 1.1 -> 1.2~git20170615.f752187-5.2, fix warning flags for GCC 14; colmap: unpin Boost (#359356)

r-ryantm committed:
phpunit: 11.5.1 -> 11.5.2

sikmir committed:
orbiton: modernize

khaneliman committed:
luaPackages: update on 2024-12-28

khaneliman committed:
vimPlugins: update on 2024-12-28

khaneliman committed:
vimPlugins.nvim-treesitter: update grammars

sikmir committed:
orbiton: 2.68.5 -> 2.68.6 (#368817)

ck3d committed:
x32edit,m32edit: 4.1 -> 4.3 (#368081)

ck3d committed:
nak: 0.7.6 -> 0.9.1 (#368032)

ck3d committed:
cargo-deb: 2.9.3 -> 2.9.4 (#368527)

nixpkgs-merge-bot[bot] committed:
myks: 4.2.5 -> 4.2.6 (#368695)

adamcstephens committed:
pamtester: add autoreconfHook (#368792)

marcusramberg committed:
proton-ge-bin: GE-Proton9-21 -> GE-Proton9-22 (#368787)

marcusramberg committed:
perlPackages.NetIDNEncode: fix use of function removed in Perl 5.38.0 (#368797)

DimitarNestorov committed:
grandperspective: remove `with lib;`

emilazy committed:
msgpack-cxx: 6.1.1 -> 7.0.0 (#353400)

misuzu committed:
3proxy: fixup build with gcc14 (#367714)

misuzu committed:
ryujinx-greemdev: 1.2.76 -> 1.2.78 (#368059)

gepbird committed:
bruno: disable telemetry

nixpkgs-merge-bot[bot] committed:
phpunit: 11.5.1 -> 11.5.2 (#368840)

Pandapip1 committed:
libdwg: remove

Pandapip1 committed:
wivrn: remove unused libdwg dependency

IliyanKostov9 committed:
Update package.nix

sikmir committed:
nchat: 5.3.5 -> 5.4.2 (#368821)

nixpkgs-merge-bot[bot] committed:
clusternet: 0.17.2 -> 0.17.3 (#368816)

GaetanLepage committed:
distant: init at 0.20.0

r17x committed:
ocamlPackages.h2: init h2-lwt, h2-lwt-unix at 0.13.0

marcin-serwin committed:
cuneiform: add patch for gcc 14

kleifgch committed:
nextcloud-client: remove unneeded patch

The removed patch was intended to install the dbus service in the proper
location, but due to a typo (`service` vs. `services`) never did. In the
meantime, upstream introduced commit [1] making the patch unnecessary.

To check that everything is in order, one can build nextcloud-client and
verify that the service file is located at $out/share/dbus-1/services/.

[1] https://github.com/nextcloud/desktop/commit/6e1e8a8bdc1a8a1b9894792cc3e3664065dfda39

thiagokokada committed:
uasm: mark as broken in darwin

doronbehar committed:
taskwarrior3: fix version and add versionCheckHook test (#368520)

thiagokokada committed:
libretro.mupen64plus: add patches for GCC14 (#368796)

GaetanLepage committed:
grandperspective: 3.4.2 -> 3.5.1, add update script (#368605)

r-ryantm committed:
openapi-python-client: 0.22.0 -> 0.23.0

linsui committed:
fdroidserver: 2.3.1 -> 2.3.4

Commit:
vscode: 1.96.0 -> 1.96.2

nixpkgs-merge-bot[bot] committed:
pik: 0.12.0 -> 0.13.0 (#368823)

nixpkgs-merge-bot[bot] committed:
code-cursor: 0.44.5 -> 0.44.9 (#368768)

r-ryantm committed:
victoriametrics: 1.108.0 -> 1.108.1

r-ryantm committed:
trufflehog: 3.87.0 -> 3.88.0

jnsgruk committed:
rockcraft: 1.6.0 -> 1.7.0 (#368735)

mweinelt committed:
home-assistant-custom-lovelace-modules.hourly-weather: 6.4.0 -> 6.5.0 (#368818)

r-ryantm committed:
srtrelay: 1.1.0 -> 1.3.0

r-ryantm committed:
home-assistant-custom-components.xiaomi_gateway3: 4.0.7 -> 4.0.8

r-ryantm committed:
home-assistant-custom-components.xiaomi_miot: 1.0.2 -> 1.0.7

lucasew committed:
nextcloud-client: move to by-name, tweak

Signed-off-by: lucasew <lucas59356@gmail.com>

lucasew committed:
nextcloud-client: 3.14.3 -> 3.15.2

Signed-off-by: lucasew <lucas59356@gmail.com>

lucasew committed:
nextcloud-client: add libsysprof-capture to dependencies as asked by the build system

Signed-off-by: lucasew <lucas59356@gmail.com>

mweinelt committed:
home-assistant-custom-components.xiaomi_miot: 1.0.2 -> 1.0.7 (#366118)

mweinelt committed:
home-assistant-custom-components.xiaomi_gateway3: 4.0.7 -> 4.0.8 (#368913)

cafkafk committed:
normcap: fix on GNOME wayland when used via keybind or alt-f2 (#351763)

thiagokokada committed:
uasm: 2.56.2 -> 2.57  (#368803)

toonn committed:
darwin.xcode: add 16.2 (#364482)

r-ryantm committed:
changedetection-io: 0.48.01 -> 0.48.05

nicoonoclaste committed:
Merge #361424: refactor lib.packagesFromDirectoryRecursive (v2)

traxys committed:
djlint: 1.35.2 -> 1.36.4

Diff: https://github.com/Riverside-Healthcare/djlint/compare/refs/tags/v1.35.2...v1.36.4

xokdvium committed:
rockcraft: actually use sources for 1.7.0

misuzu committed:
pcmanfm: fixup build with gcc14 (#368828)

xokdvium committed:
rockcraft: add passthru.tests.version

traxys committed:
fittrackee: 0.8.10 -> 0.8.12

Diff: https://github.com/SamR1/FitTrackee/compare/refs/tags/v0.8.10...v0.8.12

Changelog: https://github.com/SamR1/FitTrackee/blob/refs/tags/v0.8.12/CHANGELOG.md

emilazy committed:
swift: source darwin-sdk-setup.bash (#360207)

GaetanLepage committed:
swiftformat: 0.47.10 -> 0.55.4 (#367077)

RossComputerGuy committed:
teams: create Android team (#368434)

arianvp committed:
nixos/networkmanager: match upstream units Install sections  (#368791)

emilazy committed:
darwin: expose `requireXcode` (#367063)

GaetanLepage committed:
nextcloud-client: fix dbus service location (#368873)

GaetanLepage committed:
djlint: 1.35.2 -> 1.36.4 (#368925)

GaetanLepage committed:
fittrackee: 0.8.10 -> 0.8.12 (#368934)

7c6f434c committed:
ipmiutil: 3.1.6 -> 3.2.0 (#368583)

fpletz committed:
srtrelay: 1.1.0 -> 1.3.0 (#368912)

nixpkgs-merge-bot[bot] committed:
cargo-show-asm: 0.2.43 -> 0.2.45 (#368839)

Saturn745 committed:
netbird: 0.34.1 -> 0.35.1

Diff: https://github.com/netbirdio/netbird/compare/v0.34.1...0.35.1

Changelog: https://github.com/netbirdio/netbird/releases/tag/v0.35.1

khaneliman committed:
{luaPackages,vimPlugins}: update on 2024-12-28 (#368843)

khaneliman committed:
neovim-require-check-hook: include check inputs; vimPlugins: nativeCheckInputs -> checkInputs (#367802)

emilylange committed:
apko: 0.21.0 -> 0.22.4 (#367677)

getchoo committed:
treewide: add updateScript to gnome circle packages (#367823)

* decoder: remove gnome circle team from maintainers

This was mistakenly placed here instead of in `gnome-decoder`

* gnome-decoder: add gnome circle team to maintainers

* treewide: add updateScript to gnome circle packages

GaetanLepage committed:
fdroidserver: 2.3.1 -> 2.3.4 (#364927)

r-ryantm committed:
bilibili: 1.16.1-1 -> 1.16.1-2

GaetanLepage committed:
radicale: add ldap3 dependency to fix ldap support (#360617)

GaetanLepage committed:
nextcloud-client: 3.14.3 -> 3.15.2, to by-name, minor tweaks (#368914)

Scrumplex committed:
sofia-sip: fix build with gcc-14 (#368610)

Scrumplex committed:
libdwg: remove (#368858)

mattpolzin committed:
bruno: 1.34.2 -> 1.37.0 (#364971)

uninsane committed:
SDL_sound: fix cross build (#367712)

fpletz committed:
gnome2.gtksourceview: fix build with gcc-14 (#368606)

fpletz committed:
ocamlPackages.ctypes-foreign: fix build with gcc-14 (#368607)

jiriks74 commented:
Another note:

Make your commits (and the PR title) fit [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions)

fabaff committed:
python312Packages.python-telegram-bot: 21.7 -> 21.9 (#368217)

Diff: https://github.com/python-telegram-bot/python-telegram-bot/compare/refs/tags/v21.7...v21.9

Changelog: https://github.com/python-telegram-bot/python-telegram-bot/blob/v21.9/CHANGES.rst

JohnRTitor committed:
vscode: 1.96.0 -> 1.96.2 (#368895)

Samasaur1 committed:
jujutsu: update owner; change from rev to tag

nixpkgs-merge-bot[bot] committed:
bilibili: 1.16.1-1 -> 1.16.1-2 (#368956)

Gliczy committed:
gzdoom: 4.13.2 -> 4.14.0

drupol committed:
top-level/aliases: fix buildFHSUserEnv deprecation message typo (#368661)

khaneliman committed:
xorg.xorgserver: fix darwin build (#366504)

Bot-wxt1221 committed:
arc_unpacker: fix build

gador committed:
python31{2,3}Packages.deal: disable failing tests (#368407)

wineee committed:
qtcreator: pin llvmPackages 18 (#368736)

wineee committed:
lucenepp: fix build with boost 1.85.0 (#368744)

GaetanLepage committed:
distant: init at 0.20.0 (#368866)

vcunat committed:
arc_unpacker: fix build (#369012)

fabaff committed:
trufflehog: 3.87.0 -> 3.88.0 (#368898)

GaetanLepage committed:
xcodegen: init at 2.42.0 (#367199)

nixpkgs-merge-bot[bot] committed:
monetdb: 11.51.3 -> 11.51.7 (#368510)

fabaff committed:
python312Packages.python-roborock: 2.8.3 -> 2.8.4 (#368714)

fabaff committed:
python312Packages.aiogram: 3.15.0 → 3.16.0 (#367472)

JohnRTitor committed:
koboldcpp: revert e62a47c `add (darwinMinVersionHook "10.15")`, 1.80.1 -> 1.80.3 (#368702)

GaetanLepage committed:
python311Packages.jenkins-job-builder: 6.4.1 -> 6.4.2 (#367285)

fabaff committed:
python312Packages.ha-mqtt-discoverable: 0.16.2 -> 0.16.4 (#368710)

GaetanLepage committed:
duplicati: 2.0.8.1 -> 2.1.0.2 (#367283)

matthiasbeyer committed:
cargo-semver-checks: 0.34.0 -> 0.38.0 and refactor (#368838)

Scrumplex committed:
oscavmgr: add version test (#365419)

nixpkgs-merge-bot[bot] committed:
changedetection-io: 0.48.01 -> 0.48.05 (#366132)

wegank committed:
kdePackages.drumstick: 2.9.1 -> 2.10.0 (#367632)

drupol committed:
roboto: patch `RobotoCondensed` font to fix `usWidthClass` (#368615)

r-ryantm committed:
mympd: 19.0.1 -> 19.0.2

Scrumplex committed:
godini: init at 1.0.0, treegen: init at 1.1.0 (#368704)

drupol committed:
Bump and rename siduck76-st > st-snazzy (#360236)

getchoo committed:
ghostty: add nixos tests, add build options, fix x11 backend (#368726)

* ghostty: add nixos test

* ghostty: add nixosTests.allTerminfo to passthru.tests

* ghostty: factor out dependencies

This is meant to make cross platform support a bit easier. The options
are kept private as they aren't meant to be touched by end users

* ghostty: add optimizationLevel option

* ghostty: cleanup outputs

* ghostty: fix x11 backend

Forcing linkage isn't enough for Zig's `dlopen()` call. Let's just point
it towards the exact path instead

* ghostty: add darwin to meta.platforms

drupol committed:
jujutsu: update owner; change from rev to tag (#368985)

flokli committed:
nixos/movim: fixups (#364229)

flokli committed:
rl-2411: add an entry for the new kmonad module (#368834)

doronbehar committed:
mympd: 19.0.1 -> 19.0.2 (#369045)

nixpkgs-merge-bot[bot] committed:
openapi-python-client: 0.22.0 -> 0.23.0 (#368889)

misuzu committed:
nanovna-saver: 0.6.5 -> 0.6.8 (#366881)

misuzu committed:
chirp: 0.4.0-unstable-2024-12-16 -> 0.4.0-unstable-2024-12-17 (#367477)

misuzu committed:
nushellPlugins.units: 0.1.3 -> 0.1.4 (#367724)

misuzu committed:
netbird: 0.34.1 -> 0.35.1 (#368945)

misuzu committed:
enzyme: 0.0.168 -> 0.0.170 (#368158)

misuzu committed:
linuxPackages.corefreq: 1.98.7 -> 2.0.0 (#368186)

GaetanLepage committed:
peakperf: init at 1.17-unstable-2024-10-07 (#366989)

misuzu committed:
vscode-extensions.chenglou92.rescript-vscode: 1.58.0 -> 1.60.0 (#368253)

misuzu committed:
mainsail: 2.13.1 -> 2.13.2 (#368450)

misuzu committed:
signal-cli: 0.13.10 -> 0.13.11 (#368457)

misuzu committed:
python312Packages.manifestoo-core: 1.8.1 -> 1.8.2 (#368472)

misuzu committed:
air: 1.61.4 -> 1.61.5 (#368548)

misuzu committed:
python312Packages.coinmetrics-api-client: 2024.12.11.19 -> 2024.12.23.19 (#368386)

misuzu committed:
rain: 1.19.0 -> 1.20.2 (#365754)

misuzu committed:
python312Packages.lib4sbom: 0.7.5 -> 0.8.1 (#365565)

GaetanLepage committed:
gpufetch: init at 0.25 (#366983)

misuzu committed:
python312Packages.glfw: 2.7.0 -> 2.8.0 (#358230)

misuzu committed:
nixos/wireguard-networkd: fix loading pre shared keys for peers without a custom name (#368684)

yorickvP committed:
victoriametrics: 1.108.0 -> 1.108.1 (#368893)

azahi committed:
gzdoom: 4.13.2 -> 4.14.0 (#368993)

jnsgruk committed:
rockcraft: actually use sources for 1.7.0, add passthru.tests.version (#368933)

yorickvP committed:
mealie: 2.2.0 -> 2.3.0 (#361059)

7c6f434c committed:
cuneiform: add patch for gcc 14 (#368870)

IliyanKostov9 committed:
Merge branch 'NixOS:master' into patch-2

IliyanKostov9 committed:
Update package.nix
